cli/js/lib.deno.ns.d.ts

```diff
@@ -71,6 +71,12 @@ declare namespace Deno {
   /** Exit the Deno process with optional exit code. */
   export function exit(code?: number): never;
 
+  /** Get process priority */
+  export function getPriority(pid?: number): number;
+
+  /** Set process priority */
```

Comment:
As above, there are some nuances missing in this description I think.  Here is Node's API documentation for this:

> Attempts to set the scheduling priority for the process specified by pid. If pid is not provided or is 0, the process ID of the current process is used.
>
>The priority input must be an integer between -20 (high priority) and 19 (low priority). Due to differences between Unix priority levels and Windows priority classes, priority is mapped to one of six priority constants in os.constants.priority. When retrieving a process priority level, this range mapping may cause the return value to be slightly different on Windows. To avoid confusion, set priority to one of the priority constants.
>
>On Windows, setting priority to PRIORITY_HIGHEST requires elevated user privileges. Otherwise the set priority will be silently reduced to PRIORITY_HIGH.

Reply:
idem, on my way to do it